bazel/repository_locations.bzl

```diff
@@ -10,24 +10,27 @@ REPOSITORY_LOCATIONS = dict(
     ),
     com_github_apache_thrift = dict(
         sha256 = "7d59ac4fdcb2c58037ebd4a9da5f9a49e3e034bf75b3f26d9fe48ba3d8806e6b",
-        urls = ["https://files.pythonhosted.org/packages/c6/b4/510617906f8e0c5660e7d96fbc5585113f83ad547a3989b80297ac72a74c/thrift-0.11.0.tar.gz"],  # 0.11.0
         strip_prefix = "thrift-0.11.0",
+        urls = ["https://files.pythonhosted.org/packages/c6/b4/510617906f8e0c5660e7d96fbc5585113f83ad547a3989b80297ac72a74c/thrift-0.11.0.tar.gz"],
     ),
     com_github_bombela_backward = dict(
-        commit = "44ae9609e860e3428cd057f7052e505b4819eb84",  # 2018-02-06
-        remote = "https://github.com/bombela/backward-cpp",
+        sha256 = "ad73be31c5cfcbffbde7d34dba18158a42043a109e7f41946f0b0abd589ed55e",
```

Comment:
Super awesome, I think even without tags this is much clearer. One quick check; did you verify the `sha256` validation is working here as intended?

Reply:
I did check the validation and it is working. I changed a bunch of sha's to validate and they fail on build as intended. I also had an incorrect sha for gcovr and pushed a commit to fix. So I think this is ready.